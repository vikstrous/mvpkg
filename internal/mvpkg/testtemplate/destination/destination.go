package destination

import "example.com/source/testpkg"

func exampleFunc() {
	testpkg.ExampleFunc()
}
