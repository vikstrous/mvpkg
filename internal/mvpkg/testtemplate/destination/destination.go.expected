package destination

import "example.com/destination/testpkg"

func exampleFunc() {
	testpkg.ExampleFunc()
}
